controllers/metal3.io/baremetalhost_controller_test.go

```diff
@@ -764,8 +764,6 @@ func TestSecretUpdateOwnerRefAndEnvironmentLabelOnStartup(t *testing.T) {
 	secret = getHostSecret(t, r, host)
 	assert.Equal(t, host.Name, secret.OwnerReferences[0].Name)
 	assert.Equal(t, "BareMetalHost", secret.OwnerReferences[0].Kind)
-	assert.True(t, *secret.OwnerReferences[0].Controller)
```

Comment:
Could switch this to `assert.False` rather than deleting the test.

Reply:
See a comment below.